src/Microsoft.ML.Data/DataDebuggerPreview.cs

```diff
@@ -94,14 +94,14 @@ internal RowInfo(int n)
             }
         }
 
-        public sealed class ColumnInfo
+        public sealed class ColumnOptions
```

Comment:
>ColumnOptions [](start = 28, length = 13)

Why?

Reply:
+1 :)

---
In reply to: [259962854](https://github.com/dotnet/machinelearning/pull/2709#discussion_r259962854) [](ancestors = 259962854)

Reply:
Will revert that.